### DISCUSSION TITLE
Error Installing Jupyter Notebook

### DISCUSSION THREAD

**Discussion by Wicaedt** [teamalgoritma/community#496] [Q&A] [answered]
Saya baru saja menginstall packages baru untuk environment baru dan mengalami kenda saat instalasi kernel tadi. Tetapi sudah berhasil karena saya sudah coba mengikuti arahan dari anaconda prompt tersebut. Tapi setelah berhasil, command 'jupyter notebook' tidak bisa terlaksana dengan notifikasi sebagai berikut.

![image](https://user-images.githubusercontent.com/63247847/208586477-30281368-58af-4af4-aac7-3e9379a53a9a.png)

Untuk perhatian dan bantuannya saya ucapkan terima kasih.

**Comment by yosiaazarya** [comment, marked as answer]
Untuk menjalankan jupyter notebook, Bapak dapat berpindah terlebih dahulu ke virtual environment base dengan cara `conda activate base`
lalu kemudian mengetikkan `jupyter notebook`. Semoga dapat membantu!
